core/src/main/java/tc/oc/pgm/api/map/Gamemode.java

```diff
@@ -1,6 +1,7 @@
 package tc.oc.pgm.api.map;
 
 public enum Gamemode {
+  FIVE_CONTROL_POINT("5cp", "5 Control Points", "5CP"),
```

Comment:
gamemodes are sorted alphabetically, `FIVE` should go between FREE_FOR_ALL and FLAG_FOOTBALL

Reply:
They seem to be alphabetized by ID, for some reason — `DEATHMATCH` is after `SURVIVAL_GAMES`, which I assumed was because of `tdm` going after `sg`

Reply:
yeah... i think sorting by enum constant makes more sense

Reply:
Done — apparently there were many others that got rearranged slightly, that I hadn't noticed before.